examples/event_handling/test_mouseclicks.py

```diff
@@ -1,32 +0,0 @@
-from __future__ import print_function
```

Comment:
Isn't that file autogenerated from the example? It's not in version control as far as I can see